packages/typespec-client-generator-core/test/decorators/alternate-type.test.ts

```diff
@@ -160,6 +160,102 @@ describe.each([
   });
 });
 
+it.each([
+  ["string[]", "utcDateTime[]"],
+  ["int64[]", "utcDateTime[]"],
+  ["duration[]", "string[]"],
+])(
```

Comment:
Please make sure this work: original type is union and alternate type is string, like
```
model A {  
  property: SomeUnion;
}

union SomeUnion {
  string,
  SCSI: "SCSI",
  NVMe: "NVMe",
}
@@alternateType(A.property,
  string
);
```

Reply:
> union SomeUnion {
>   string,
>   SCSI: "SCSI",
>   NVMe: "NVMe",
> }

when the source is model property, alternate can be any type.